client/elements/sc-page-dictionary.html

```diff
@@ -167,20 +167,14 @@ <h1><span class="dictionary-results-description">Definitions for</span> <span
           <div class="related-terms sc-scrollbar">
             <h3>Adjacent Terms</h3>
             <ul class="near-terms">
-              <template is="dom-repeat" items="[[dictonaryAdjacent]]">
-                <li class$="[[_calculateClass(item)]]">
-                  <a href="/define/[[item]]">[[item]]</a>
-                  <!-- <template is="dom-if" if="[[item.trans]]"><em>([[item.trans]])</em></template> -->
-                </li>
+              <template is="dom-repeat" items="{{dictionaryAdjacentArray}}">
```

Comment:
I think `dictionaryAdjacentArray` should be using one-way data-binding (the [[ ]] operator) instead of {{ }}. That allows for performance improvements under the hood. Unless the data is being modified by the child of course.